### PR TITLE
add service topic to cli

### PR DESCRIPTION
**Pull request by rolyatmax**
Starting to support the concept of services with the jamsocket cli.

Still some work to be done here, which I'll point out in comments.

### CODE REVIEW

File: src/commands/service/create.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { Command } from '@oclif/core'
+import { request, readJamsocketConfig, API, SERVICE_CREATE_ENDPOINT } from '../../common'
+
+export default class Create extends Command {
+  static description = 'Creates a service'
+
+  static examples = [
+    '<%= config.bin %> <%= command.id %>',
```
Comment (paulgb) [inline src/commands/service/create.ts]: I might be misreading the purpose of `examples`, but is this supposed to be an example invocation? If so I guess it should have a service name.

File: src/commands/service/create.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      headers: {
+        'Authorization': `Basic ${auth}`,
+        'Content-Type': 'application/json',
+      },
+    })
+
+    // TODO: Handle creation errors & authentication errors
+    // const response = JSON.parse(responseBody)
+    this.log(`response from ${endpoint}:`)
+    this.log(responseBody)
```
Comment (rolyatmax) [inline src/commands/service/create.ts]: Should we have some sort of structured response returned from this endpoint? I'm okay with using HTTP status codes for status, but I feel like we should return a message too - maybe in JSON? Particularly for errors (like "service name already exists").

File: src/commands/service/create.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    const endpoint = `${API}${SERVICE_CREATE_ENDPOINT}`
+    const responseBody = await request(endpoint, body, {
+      method: 'POST',
+      headers: {
+        'Authorization': `Basic ${auth}`,
+        'Content-Type': 'application/json',
+      },
+    })
+
+    // TODO: Handle creation errors & authentication errors
```
Comment (paulgb) [inline src/commands/service/create.ts]: The API now (as of committed but non-prod changes) uses a standard error format across all endpoints, we can discuss more in person. Not for this PR but I think we can refactor some of this into a standard flow of constructing an API request, sending it, and handling an error result.

Another advantage of that approach is that we could have an alternate flow that takes the same command but outputs a `curl` command.

Comment (rolyatmax) [reply src/commands/service/create.ts]: Ahh gotcha! Sounds good - we can chat more tomorrow about it!

File: src/commands/service/create.ts (hunk 1 above)
Comment (rolyatmax) [reply src/commands/service/create.ts]: Yeah I need to add one - good catch!

**Review by paulgb — APPROVED** [review]
Looks good, I'm going to go ahead and merge this to open a new PR with my changes on top of it.

**Pull request merged**